### DISCUSSION TITLE
dragging Node

### DISCUSSION THREAD

**Discussion by elmanAliev** [xyflow/xyflow#3496] [Q&A]
Hello! I am facing problem with dragging Node in reactflow v 11.9.3. I am using nodes and edges   props for ReactFlow so when i am dragging my node with <div class="react-flow__node react-flow__node-defaultNode nopan selected selectable" data-testid="rf__node-34212"

I am seeing that it's inline styles are not updating. So in the end i only see the final position but not the process of dragging.

Do you know how to fix it?

**Comment by bcakmakoglu** [comment]
There's a bit of information required to help you out here:
Are you using a *controlled* or *uncontrolled* flow?

If it's *controlled* how are you applying the changes provided by React Flow?

**Reply by bcakmakoglu** [reply]
Maybe a little bit more of your code (or an example of it) would be helpful as well :)

**Comment by elmanAliev** [comment]
my question has changed a bit. I realized that dragging a node doesn't work because I changed "defaultNodes" to "nodes" in my code. 
why did I decide to change? because after I migrated from reactflow version 10 to version 11, "defaultNodes" are not displayed, and "nodes" are displayed. therefore, I have a new question, why can this happen that "defaultNodes" are not displayed?
